openfisca_nsw_safeguard/variables/ESS_PDRS_Estimator/F17/activity_eligibility/F17_activity_eligibility_parameters.py

```diff
@@ -0,0 +1,143 @@
+import numpy as np
+from openfisca_core.variables import Variable
+from openfisca_core.periods import ETERNITY
+from openfisca_core.indexed_enums import Enum
+from openfisca_nsw_base.entities import Building
+
+
+class F17_equipment_installation(Variable):
+    value_type = bool
+    entity = Building
+    default_value = True
+    definition_period = ETERNITY
+    metadata = {
+      'display_question' : 'Is the equipment being installed an air source heat pump water heater as defined by AS/NZS 4234?',
+      'sorting' : 1,
+      'eligibility_clause' : """In ESS F17 Equipment Requirements Clause 1 it states that the New End-User Equipment must be an air source heat pump water heater as defined by AS/NZS 4234."""
+    }
+
+
+class F17_installed_by_qualified_person(Variable):
+    value_type = bool
+    entity = Building
+    default_value = True
+    definition_period = ETERNITY
+    metadata = {
+      'display_question' : 'Has installation of the new equipment been performed or supervised by a suitably licensed person?',
+      'sorting' : 2,
+      'eligibility_clause' : """In ESS F17 Implementation Requirements Clause 2 it states that the activity must be performed or supervised by a suitably qualified licence holder in compliance with the relevant standards and legislation."""
+    }
+    
+
+class F17_minimum_payment(Variable):
+    value_type = bool
+    entity = Building
+    default_value = True
+    definition_period = ETERNITY
+    metadata = {
+      'display_question' : 'Are you aware that you are required to make a minimum payment towards the cost of your upgrade?',
+      'sorting' : 3,
+      'eligibility_clause' : """In ESS Clause 9.9.1E it states that the Accredited Certificate Provider has evidence satisfactory to the Scheme Administrator that the Purchaser has paid for the Implementation, assessment and other associated works carried out at the Site a Net Amount of at least $200 (excluding GST) for each item of End-User Equipment installed as part of an Implementation using any of Activity Definitions F1.1, F1.2, F16 or F17."""
+    }
+
+
+class F17_building_BCA_not_class_1_or_4(Variable):
+    value_type = bool
+    entity = Building
+    default_value = True
```

Comment:
This needs to default as False to be eligible, could you please update (and fix tests as well)

Reply:
this has been updated and fixed